### PR TITLE
Namespacing config

### PR DESCRIPTION
**Pull request by slemrmartin**
**Issue** #8 

Changed LoadedConfig to `AppCommonRuby::Config.load`

### CODE REVIEW

File: lib/app-common-ruby/config.rb [hunk 1]
```diff
@@ -2,43 +2,65 @@
 require 'json'
 require_relative 'types'
 
-arg_config = ENV['ACG_CONFIG'] || 'test.json'
+module AppCommonRuby
```
Comment (slemrmartin) [inline lib/app-common-ruby/config.rb:5]: just added module, no other change in this file

File: lib/app-common-ruby/types.rb [hunk 2]
```diff
@@ -1,346 +1,348 @@
...
-  attr_accessor :privateEndpoints
-
-  def initialize(attributes)
-    super
-    raise 'The input argument (attributes) must be a hash' if (!attributes || !attributes.is_a?(Hash))
-
-    attributes = attributes.each_with_object({}) do |(k, v), h|
-      raise "The input [#{k}] is invalid" unless valid_keys.include?(k.to_sym)
-      h[k.to_sym] = v
+module AppCommonRuby
```
Comment (slemrmartin) [inline lib/app-common-ruby/types.rb:4]: just added module, no other change in this file

**Comment by slemrmartin** [comment]
@hsong-rh please review. 

This change will need deployment to rubygems as apps pointing to master will be broken

**Comment by slemrmartin** [comment]
@mkanoor please review

**Review by mkanoor — APPROVED** [review]

File: lib/app-common-ruby/types.rb (hunk 2 above)
Comment (hsong-rh) [reply lib/app-common-ruby/types.rb:4]: @slemrmartin This file is autogenerated (marked in comments) by https://github.com/RedHatInsights/app-common-ruby/blob/master/bin/ruby_class_converter.rb, cannot change directly.

**Review by hsong-rh — APPROVED** [review]

**Pull request merged**